test/specs/ExpectNoAttributeValueSpec.ts

```diff
@@ -3,27 +3,39 @@ import { BrowserUtils } from '../../src/commons/BrowserUtils';
 import { describeCommon } from '../TestHelper';
 
 export namespace PageLocator {
-  export const FORM: string = "//*[@id='forms']/form";
+  export const EMPTY_DIV: string = '//*[@id="noAttDiv"]';
 }
 /**
  * wdio-allure-ts ExpectNoAttributeValueSpec action test
  */
 describeCommon('expectNoAttributeValue', () => {
   it("Doesn't contains value", () => {
     expect(() =>
-      BrowserUtils.expectNoAttributeValue(PageLocator.FORM, 'method', 'pos')
+      BrowserUtils.expectNoAttributeValue(
```

Comment:
consider adding test on not existing element

Reply:
done

Reply:
not working properly removing

Reply:
what's not working properly?

Reply:
Chai expect doesn't catch the error